### DISCUSSION TITLE
Accessing data inside a property from REST Endpoint

### DISCUSSION THREAD

**Discussion by claudioriginio** [Budibase/budibase#2787] [Help] [answered]
Hi there

my data source (Rest API) doesn’t return the plain array, but it returns it in a property.
How can I select the rows from the “rows” property in the body?

{
  "rows": [
    {
      "_id": "Dz5W7dYvRtyIr1jqrcvMbw",
      "_mtime": "2021-08-19T15:15:16.180+00:00",
      "_ctime": "2021-07-27T15:17:16.737+00:00",
      "Name": "XXXX",
      "Typ": "Input",
    },
    {
      "_id": "S2l7MdWrRLykVrG7JiYK1w",
      "_mtime": "2021-07-27T16:01:52.701+00:00",
      "_ctime": "2021-07-27T15:20:08.180+00:00",
      "Name": "YYYY",
      "Typ": "Input",
    },
    {
      "_id": "exPjbfheR4er6G0gbacpiw",
      "_mtime": "2021-07-27T16:01:55.537+00:00",
      "_ctime": "2021-07-27T15:31:19.737+00:00",
      "Name": "ZZZZ",
      "Typ": "Input",
    },
...
....

The only option I have is:

![image](https://user-images.githubusercontent.com/9660495/135247031-f57d83c5-6fbb-49d8-9018-ad5847a32a4c.png)

Thanks for your help

**Comment by mjashanks** [comment, marked as answer]
Hey @claudioriginio - unfortunately, this is a known limitation. We are addressing this with our release at the end of October. Then, you will be able to add a "Transformer" - whereby you can write javascript to modify the response.

Until then, your only option is to set up an intermediary API that does the transformation.

**Reply by claudioriginio** [reply]
Thank you @mjashanks, therefore I'll wait for the next release.
